src/main/java/com/onarandombox/MultiverseCore/utils/WorldManager.java

```diff
@@ -707,16 +714,23 @@ private boolean isMVWorldAlias(final String alias) {
     public void loadDefaultWorlds() {
         this.ensureConfigIsPrepared();
         List<World> myWorlds = this.plugin.getServer().getWorlds();
+        this.defaultWorldNames = new HashSet<>(myWorlds.size());
         for (World w : myWorlds) {
             String name = w.getName();
             if (!worldsFromTheConfig.containsKey(name)) {
                 String generator = null;
                 if (this.defaultGens.containsKey(name)) {
                     generator = this.defaultGens.get(name);
                 }
-                this.addWorld(name, w.getEnvironment(), String.valueOf(w.getSeed()), w.getWorldType(), w.canGenerateStructures(), generator);
+                if (!this.addWorld(name, w.getEnvironment(), String.valueOf(w.getSeed()), w.getWorldType(), w.canGenerateStructures(), generator)) {
```

Comment:
`String.valueOf` -> `Long.toString`

Reply:
This makes no difference as far as I am aware. They will both result in the same thing (please correct me if I'm wrong).

Reply:
String.valueOf is just using the ToString method. Long.ToString _may_ be different. But isnt part of the change anyways, was String.valueOf.

Reply:
`String.valueOf` returns the same when you call `Long#toString`. Yea, obviously does not makes any difference, just making sure that we're not calling two methods.